gallery/visualtests/testUtils.js

```diff
@@ -56,15 +56,37 @@ module.exports = {
 
     beforeEach(async function() {
       page = await browser.newPage();
+
       await page.goto(pageUrl + pageFragmentIdentifier);
 
+      // eslint-disable-next-line no-undef
+      if (process.env.RSC_GALLERY_THEME === 'DARK') {
+        await page.emulateMediaFeatures([{ name: 'prefers-color-scheme', value: 'dark' }]);
+      }
+
       if (ignoreVersionNumber) {
         await hideVersionNumber();
       }
 
+      await page.evaluate(() => {
+        window.setThemingEnabled(true);
+      });
```

Comment:
Related to https://github.com/sonatype/sonatype-react-shared-components/pull/808/files#r1004805779 , once `removeThemeEnabling` becomes `setThemeEnabling` you can call it here instead of dealing with `page.evaluate` again

Reply:
> Related to https://github.com/sonatype/sonatype-react-shared-components/pull/808/files#r1004805779 , once `removeThemeEnabling` becomes `setThemeEnabling` you can call it here instead of dealing with `page.evaluate` again

Updated here: [5740744](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/574074445b615d7b483810cd4cb99bfa87a28506)